src/browser.ts

```diff
@@ -63,6 +63,11 @@ export interface BrowserConfig {
   addArguments?: string[];
   /** WebDriver default binary arguments to omit. */
   removeArguments?: string[];
+  /**
+   * Experimental CPU Throttling rate. (1 is no throttle, 2 is 2x slowdown,
+   * etc).
+   */
+  experimentalCPUThrottlingRate?: number;
```

Comment:
I think we should call it 'cpuThrottlingRate' even though it is experimental

Reply:
Done